doc/community-release-build.md

```diff
@@ -0,0 +1,202 @@
+# 3scale-Operator Community Release
+
+[Introduction](#introduction)  
+[Main Steps Briefly](#main-steps-briefly)  
+[Versioning Strategy](#versioning-strategy)  
+[Steps to Perform Community Release in 3scale-operator repo](#steps-to-perform-community-release-in-3scale-operator-repo)  
+[Prepare release in community-operators-prod repo](#prepare-release-in-community-operators-prod-repo)
+[References](#references)
+
+
+## Introduction
+* Community Release created in parallel with Minor Product Releases or patch releases, and not for Major release.
+  * Community Release is not required for every productized build or every product patch release.
+  * In this document: Product Releases == Product Releases or patch releases
```

Comment:
@valerymo Don't think this line is required. Product patch release is still a product release? Do we need to make the distinction?

Reply:
removed. Thank you